src/vm/unixlowmem.h

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ===========================================================================
+
+#ifndef _UNIXLOWMEM_H_
+#define _UNIXLOWMEM_H_
+
+#ifdef FEATURE_PAL
```

Comment:
I would prefer including the header conditionally and removing the FEATURE_PAL here.